partcad-cli/src/partcad_cli/click/commands/status.py

```
@@ -32,35 +42,40 @@ def get_size(start_path="."):
 
 
 def get_total():
-    with logging.Action("Status", "total"):
-        total = (get_size(path)) / 1048576.0
-        logging.info("Total internal data storage size: %.2fMB" % total)
+    with logging.Action("Status", "total", transaction=transaction):
+        total = float(get_size(path))
+        logging.info("Total internal data storage size: %s" % humanfriendly.format_size(total))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)